client/scss/components/_help-block.scss

```diff
@@ -16,7 +16,7 @@
   }
 
   a {
-    color: $color-link;
+    color: theme('colors.secondary.100');
```

Comment:
@thibaudcolas  there is no secondary 200 - the figma design matches the links being secondary - 400 (which is current state).

@PaarthAgarwal I recommend we remove this change for now

Reply:
![Screenshot from 2022-07-26 14-55-00](https://user-images.githubusercontent.com/86092410/180973348-f64c0a29-ac09-4702-905a-6ce73af7a3e0.png)

Updated Screenshot

Reply:
I've taken the underline styles from the underline style used in our docs.